pkgs/applications/editors/edbrowse/default.nix

```diff
@@ -1,26 +1,11 @@
-{ stdenv, fetchFromGitHub, fetchpatch, duktape, curl, pcre, readline, openssl, perl, html-tidy }:
+{ stdenv, fetchFromGitHub, duktape, curl, pcre, readline, openssl, perl, html-tidy }:
 
 stdenv.mkDerivation rec {
   pname = "edbrowse";
-  version = "3.7.4";
+  version = "3.7.6";
 
   buildInputs = [ curl pcre readline openssl duktape perl html-tidy ];
 
-  patches = [
-    # Fix build against recent libcurl
-    (fetchpatch {
-      url = https://github.com/CMB/edbrowse/commit/5d2b9e21fdf019f461ebe62738d615428d5db963.diff;
-      sha256 = "167q8n0syj3iv6lxrbpv4kvb63j4byj4qxrxayy08bah3pss3gky";
-    })
-  ];
-
-  postPatch = ''
```

Comment:
The patch was not needed anymore, but on the other hand this `postPatch` shell hook is still needed for the build to succeed.

Even if no patches are explicitly given, the postPatch hook runs as part of [the patch phase](https://nixos.org/nixpkgs/manual/#ssec-patch-phase) of a standard nix build.

Reply:
Ohhh I see... Fixed :sweat_smile: